site/public/js/server.js

```diff
@@ -1849,7 +1849,7 @@ $.fn.isInViewport = function() {                                        // jQuer
 
 function checkSidebarCollapse() {
     var size = $(document.body).width();
-    if (size < 1000) {
+    if (size < 1150) {
```

Comment:
Why did this number go up?   Side bar should stick around longer. 
(Yes, I'm asking for the impossible, but many of us have not memorized the symbols)

Reply:
At around 1150px, the instructor gradeables page gets too wide to fit on the screen. We have 3 options: allow the table to overflow to the side of the screen (and generate a horizontal scrollabar), switch to tablet mode, or collapse the sidebar. If we collapse the sidebar, we can push the tablet cutoff point from 1150px to 950px, which allows us to maintain desktop mode in half of a 1080p monitor (960px width). I can switch it to something else though.